### PR TITLE
Adds MP bunks and Area to Sulaco and gives PO bunks Vendors

### PR DESCRIPTION
**Pull request by Derrick2Electricboogaloo**
## About The Pull Request

Adds a new area to the Sulaco, also adds the mech computer and area for sulaco.

Picture for reference:
image
Just a retrofit for the hangar workshop.

## Why It's Good For The Game

Tivi is adding mech's. Sulaco does not have a mech bay. Now it does.

## Changelog
:cl:
add: Added Mechbay and Bunks to Sulaco
add: Gave PO bunks: Loadout, Armor and uniform supply vendors
/:cl:

### CODE REVIEW

**Review by RipGrayson — APPROVED** [review]

**Comment by novaepee** [comment]
image
image

**Comment by Derrick2Electricboogaloo** [comment]
Good catch! Did not know that.

File: .github/workflows/ci_suite.yml [hunk 1]
```diff
@@ -90,6 +90,9 @@ jobs:
           mysql -u root -proot -e 'CREATE DATABASE tg_ci_prefixed;'
       - name: Install rust-g
         run: |
+          sudo dpkg --add-architecture i386
+          sudo apt update || true
+          sudo apt install -o APT::Immediate-Configure=false libssl1.1:i386
           bash tools/ci/install_rust_g.sh
       - name: Compile and run tests
```
Comment (TiviPlus) [inline .github/workflows/ci_suite.yml:97]: trash?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_suite.yml (hunk 1 above)
Comment (Derrick2Electricboogaloo) [reply .github/workflows/ci_suite.yml:97]: Yeah for some reason I had to put that in order to make a PR, I really have no idea why. Can I safely remove the file?

Comment (TiviPlus) [reply .github/workflows/ci_suite.yml:97]: As said in disc you need to revert these changes

Comment (Derrick2Electricboogaloo) [reply .github/workflows/ci_suite.yml]: It seems to have worked doing it manually with your code review.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**